src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/grouped/UnconfirmedAttendanceReportITest.java

```diff
@@ -0,0 +1,252 @@
+                getTableDataItem("041500008", "CName8", "CSurname8", "415240101", "111111", "09:00:00", "17:30:00"),
+                getTableDataItem("041500009", "CName9", "CSurname9", "415240101", "111111", "09:00:00", "17:30:00"),
+                getTableDataItem("041500017", "CName17", "CSurname17", "415240103", "111111", "09:00:00", null),
+                getTableDataItem("041500018", "CName18", "CSurname18", "415240103", "111111", "09:00:00", null),
+                getTableDataItem("041500019", "CName19", "CSurname19", "415240103", "111111", "09:00:00", null),
+                getTableDataItem("041500020", "CName20", "CSurname20", "415240103", "111111", "09:00:00", null)
+            ))
+            .add(today.concat(",").concat("CIVIL COURT"), List.of(
+                getTableDataItem("041500011", "CName11", "CSurname11", "415240102", "111111", "09:00:00", "17:30:00"),
+                getTableDataItem("041500013", "CName13", "CSurname13", "415240102", "111111", "09:00:00", null),
+                getTableDataItem("041500014", "CName14", "CSurname14", "415240102", "111111", "09:00:00", null)
+            ))
+            .add(yesterday.concat(",").concat("CROWN COURT"), List.of(
+                getTableDataItem("041500001", "CName1", "CSurname1", "415240101", "111111", "09:30:00", "17:30:00"),
+                getTableDataItem("041500002", "CName2", "CSurname2", "415240101", "111111", "08:00:00", "17:30:00"),
+                getTableDataItem("041500003", "CName3", "CSurname3", "415240101", "111111", "09:00:00", "17:30:00"),
+                getTableDataItem("041500004", "CName4", "CSurname4", "415240101", "111111", "09:00:00", "17:30:00"),
+                getTableDataItem("041500005", "CName5", "CSurname5", "415240101", "111111", "09:00:00", "17:30:00"),
+                getTableDataItem("041500010", "CName10", "CSurname10", "415240101", "111111", "09:00:00", "17:30:00"),
+                getTableDataItem("041500017", "CName17", "CSurname17", "415240103", "111111", "11:20:00", "17:30:00"),
+                getTableDataItem("041500018", "CName18", "CSurname18", "415240103", "111111", "09:00:00", "17:30:00"),
+                getTableDataItem("041500019", "CName19", "CSurname19", "415240103", "111111", "09:00:00", "17:30:00"),
+                getTableDataItem("041500020", "CName20", "CSurname20", "415240103", "111111", "09:00:00", "17:30:00")
+            ))
+            .add(yesterday.concat(",").concat("CIVIL COURT"), List.of(
+                getTableDataItem("041500011", "CName11", "CSurname11", "415240102", "111111", "09:30:00", "17:30:00"),
+                getTableDataItem("041500012", "CName12", "CSurname12", "415240102", "111111", "09:30:00", "17:30:00"),
+                getTableDataItem("041500013", "CName13", "CSurname13", "415240102", "111111", "06:30:00", "17:30:00")
+            ));
+    }
+
+    private ReportLinkedMap<String, Object> getTableDataItem(String... jurorData) {
```

Comment:
As the data is fixed we should use named variables here such as

String jurorNumber,
String firstName,
etc

Reply:
updated